csharp/src/IceGrid/README.md

```diff
@@ -1 +1,12 @@
-# TODO
+# IceGrid
+
```

Comment:
I would say it's more about IceGrid deployment management. The IceGrid locator can be used independently.